include/cucumber-cpp/internal/CukeDll.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef CUKE_CUKEDLL_HPP_
+#define CUKE_CUKEDLL_HPP_
+
+#ifndef CUKE_API_
+#if CUKE_LINKED_AS_SHARED_LIBRARY
+#define CUKE_API_ __declspec(dllimport)
+#elif CUKE_CREATE_SHARED_LIBRARY
+#define CUKE_API_ __declspec(dllexport)
+#else
+#define CUKE_API_
```

Comment:
FYI: GenerateExportHeader would call this `CUCUMBER_CPP_EXPORT` (and for Windows `#define` that to be dllimport when importing).